cmd/fl/main.go

```diff
@@ -44,12 +46,14 @@ func main() {
 
 	logger, err := buildLogger()
 
+	writer := os.Stdout
```

Comment:
Instead of using a writer bound to kong to use as input in the fn cmds to print messages, the already present logger should be used. Logger.Info (Infof) should be used to print user-facing messages, the logger also has a spinner (StartSpinner, SpinnerMessage, StopSpinner) to handle messages with an animation.

Please use that. Just remove the writer and change the fn commands input to get the logger instead of the writer. You can look into the admin command and see how to do it.